src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const listOfDataAboutPopulation = document.querySelectorAll('.population');
+let totalPopulation = 0;
+
+for (let i = 0; i < listOfDataAboutPopulation.length; i++) {
+  totalPopulation += parseInt(
+    listOfDataAboutPopulation[i].innerText.split(',').join('')
+  );
+}
+
+document.querySelector('.total-population').innerHTML = totalPopulation;
```

Comment:
You can add .toLocaleString('en-US');  in the end .It`ll set the correct commas in numbers